app/messaging/message-base.js

```diff
@@ -16,7 +16,11 @@ class MessageBase {
     } else if (this.config.connectionString) {
       this.sbClient = new ServiceBusClient(this.config.connectionString)
     } else {
-      this.sbClient = new ServiceBusClient(`Endpoint=sb://${this.config.host}/;SharedAccessKeyName=${this.config.username};SharedAccessKey=${this.config.password}`)
+      var useDevelopmentEmulator=false;
```

Comment:
Should be linted using `StandardJS` or `neostandard` -> https://defra.github.io/software-development-standards/standards/javascript_standards/#use-neostandard-to-lint-your-code

No use of `var`, should use `let` instead and no semi colons.  Spacing also needs updating.